macropad/code/keys.py

```diff
@@ -0,0 +1,197 @@
+'''
+YOU CAN EDIT THIS FILE!
+
+This is where you tell the macro pad which buttons do what.
+
+The macro pad has pages, where each page can have up to 8 buttons
+(corresponding to the 8 buttons on the macro pad).
+
+These are defined in the "pages" variable below. Each page has
+a name that will be displayed on the screen, and then a list
+of buttons. Each button has a name and an action. So in the
+example below, "Text Utilities" is the page name, and "Cut",
+"Copy" and "Paste" are the button names.
+
+>   pages = (
+>       (
+>           "Text Utilities", (
+>               ("Cut", (Keycode.CONTROL, Keycode.X)),
+>               ("Copy", (Keycode.CONTROL, Keycode.C)),
+>               ("Paste", (Keycode.CONTROL, Keycode.V)),
+>           )
+>       ),
+>   )
+
+You'll need to be really careful of using the right number of
+brackets and commas, otherwise the program won't know what is
+a button or a page.
+
+The first button defined in the list will be the top left key,
+and they'll progress in this order:
+
+1  2  3  4
+5  6  7  8
+
+Each key can do one of 5 different things.
+
+1. Single Key
+=============
+
+It can act as a single key, for example:
+
+>   ("Shoot", Keycode.ENTER),
+
+This will create a key called "Shoot" that acts exactly like
+the Enter key on a keyboard. You can hold the button down and
+get repeated presses, and when you release the button on the
+macropad, it'll stop pressing the Enter key.
+
+Take a look at
+https://docs.circuitpython.org/projects/hid/en/latest/api.html#adafruit-hid-keycode-keycode
+for the complete list of keys. 
+
+2. Several Keys
+===============
+
+It can act as several keys at once. For example:
+
+> ("Undo", (Keycode.CONTROL, Keycode.Z)),
+
+Note that the list of keys are additionally surrounded by
+brackets. That's really important.
+
+This uses the same key codes as "1. Single Key", but will
+press both those keys and won't release them until you stop
+pressing the button
+
+3. Type some text
+=================
+
+It can type a load of text for you. For example:
+
+>   ("Sign", "\n\nThanks,\nJoel"),
+
+When pressed, this will type those characters. The \n means
+a new line (i.e. press enter)
```

Comment:
Is it worth including a note of some kind not to use " in your string (or to escape it), or is that assumed knowledge?

Reply:
Added!